app/controllers/hosts_controller.rb

```diff
@@ -199,7 +199,9 @@ def externalNodes
     begin
       respond_to do |format|
         # don't break lines in yaml to support Ruby < 1.9.3
-        host_info_yaml = @host.info.to_yaml(:line_width => -1)
+        # Remove the HashesWithIndifferentAccess using 'deep_stringify_keys',
+        # then we turn it into YAML
+        host_info_yaml = @host.info.deep_stringify_keys.to_yaml(:line_width => -1)
```

Comment:
do you happen to know if there is a performance impact? I'm just wondering as this is a very common "API" endpoint.

Reply:
Quick benchmark indicates it isn't significant (`a` is a host.info):
```ruby
[7] pry(main)> Benchmark.ips do |x|
[7] pry(main)*   x.report("with deep_stringify"){a.deep_stringify_keys.to_yaml(:line_width => -1)}
[7] pry(main)*   x.report("without"){a.to_yaml(:line_width => -1)}
[7] pry(main)*   x.compare!
[7] pry(main)* end  
Warming up --------------------------------------
 with deep_stringify    21.000  i/100ms
             without    22.000  i/100ms
Calculating -------------------------------------
 with deep_stringify    213.482  (± 7.0%) i/s -      1.071k in   5.043390s
             without    215.876  (± 6.9%) i/s -      1.078k in   5.018337s

Comparison:
             without:      215.9 i/s
 with deep_stringify:      213.5 i/s - same-ish: difference falls within error

```